LoopKit/InsulinKit/NewPumpEvent.swift

```diff
@@ -10,6 +10,48 @@ import Foundation
 
 
 public struct NewPumpEvent: Equatable {
+    public static func == (lhs: NewPumpEvent, rhs: NewPumpEvent) -> Bool {
```

Comment:
This isn't quite a complete equality check. Making all the subtypes Equatable should give you this automatically. Happy to help if you need!

Reply:
Agree removed no longer needed